src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1908,6 +1913,10 @@ def aks_agentpool_update(cmd,   # pylint: disable=unused-argument
 
     if labels is not None:
         instance.node_labels = labels
+
+    if enable_custom_ca_trust is not None:
+        instance.enable_custom_ca_trust = enable_custom_ca_trust
```

Comment:
Just curious, currently we don't support to disable it once turn it on?

Reply:
We want to allow disabling it, I added another flag to facilitate that. Tests to follow